docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -764,8 +764,9 @@ Every task has some expansions available by default:
     to
 -   `${version_id}` is the id of the task's version
 -   `${workdir}` is the task's working directory
--   `${revision}` is the commit hash of the base commit of a patch or of
-    the commit for a mainline build
+-   `${revision}` is the commit hash of the base commit that a patch's changes
+    are being applied to, or of the commit for a mainline build. For PR patches,
+    the base commit value is the PR base value computed by GitHub.
```

Comment:
nit: might be confusing to describe it as a value rather than a commit, so might be clearer to just say "the base commit is the PR base chosen by GitHub".